pkg/sql/sqlbase/structured.go

```diff
@@ -3893,6 +3970,103 @@ func (desc *ColumnDescriptor) SQLString() string {
 	return f.CloseAndGetString()
 }
 
+// FormatForDisplay formats a column descriptor as a SQL string, and displays
+// user defined types in serialized expressions with human friendly formatting.
+// TODO (rohany): Maybe this should go into a new function?
+// TODO (rohany): This should have a better name?
```

Comment:
`NodeFormat`? :P (does this make it fit into one of the existing related interfaces?)
wdym by "new function"?

Reply:
There is already a `tree.Serialize` -- we have a precedent for pulling formatting with common flags into functions, and I wanted to make it clear when to use this one vs others.

Reply:
Woops, I read this comment wrong. The new function comment is stale. Perhaps `NodeFormat`!